core/include/kernel/tee_time.h

```diff
@@ -38,5 +38,7 @@ TEE_Result tee_time_get_ta_time(const TEE_UUID *uuid, TEE_Time *time);
 TEE_Result tee_time_get_ree_time(TEE_Time *time);
 TEE_Result tee_time_set_ta_time(const TEE_UUID *uuid, const TEE_Time *time);
 void tee_time_wait(uint32_t milliseconds_delay);
```

Comment:
maybe we should add a comment stating `tee_time_wait()` releases CPU trhough OP-TEE RPC means. It would help understanding the difference with `tee_time_mdelay()`.
Or maybe `tee_time_mdelay()` should be renamed `tee_time_busy_wait()`.